lib/web_ui/lib/src/engine/canvaskit/layer.dart

```diff
@@ -518,10 +533,15 @@ class ColorFilterEngineLayer extends ContainerLayer
     final CkPaint paint = CkPaint();
     paint.colorFilter = filter;
 
+    paintContext.internalNodesCanvas.save();
+    paintContext.internalNodesCanvas
```

Comment:
Can we leave a TODO to remove the clip for filters that do not affect transparent black? Clips can be expensive.